core/src/main/java/org/bitcoinj/core/TransactionDestination.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright 2022 Dash Core Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.bitcoinj.core;
+
+import com.google.common.base.Preconditions;
+import com.google.common.primitives.Ints;
+import org.bitcoinj.script.Script;
+import org.bitcoinj.script.ScriptPattern;
+
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.Arrays;
+
+/**
+ * Created by Hash Engineering on 8/25/2018.
```

Comment:
Many classes have this which doesn't seem correct :)

Reply:
Correct, it was a copy paste problem.